rllib/algorithms/marwil/marwil_rl_module.py

```diff
@@ -0,0 +1,71 @@
+import abc
+from typing import Type
+
+from ray.rllib.core.columns import Columns
+from ray.rllib.core.models.specs.specs_dict import SpecDict
+from ray.rllib.core.rl_module import RLModule
+from ray.rllib.core.rl_module.apis.value_function_api import ValueFunctionAPI
+from ray.rllib.models.distributions import Distribution
+from ray.rllib.utils.annotations import ExperimentalAPI, override
+
+
+@ExperimentalAPI
+class MARWILRLModule(RLModule, ValueFunctionAPI, abc.ABC):
```

Comment:
Nice!!

Reply:
Yeah, I wanted to make use of it immediately - while PPO is still not using it :)

Reply:
Quick fix :)